### PR TITLE
1904-10-31

### PR DESCRIPTION
**Pull request by JuliaCavaliere**
Deliverable 1 (pages 1-3)

### CODE REVIEW

File: 1904-10-31.xml [hunk 1]
```diff
@@ -0,0 +1,2512 @@
...
+                    <p>The following steamers are expected to leave Port Said on or about the
+                        following dates:--</p>
+                    <p>
+                        <table rows="11" cols="7">
+                            <head/>
+                            <row>
+                                <cell>"Formosa"</cell>
+                                <cell>4,045</cell>
+                                <cell>Tons</cell>
+                                <cell><date when="1905-01-08">30</date></cell>
```
Comment (whanley) [inline 1904-10-31.xml:88]: when attributes need to match the date, thus: `when="1904-10-30"`

File: 1904-10-31.xml [hunk 2]
```diff
@@ -0,0 +1,2512 @@
...
+                    <p>George Royle, Esq… … … … … … … … … … … … … … PORT-SAID.</p>
+                    <p>Messrs. Haselden &amp; Co. … … … … … … … … … … … … ALEXANDRIA.</p>
+                    <p>F.G. DAVIDSON, Superintendant P. &amp; O. S. N. Company in Egypt. SUEZ.
+                        xx-xx-xx</p>
+                    <p/>
+                </div>
+                <div type="advert" xml:id="deg-ad-opl01">
+                    <head>Orient-Pacific Line of Royal Mail Steamers.</head>
+                    <p>OUTWARDS to AUSTRALIA.</p>
+                    <p><name>R.M.S. "Orotava"</name> will leave Suez about <date when="1905-07-28">Nov. 4</date></p>
```
Comment (whanley) [inline 1904-10-31.xml:205]: update when attributes

File: 1904-10-31.xml [hunk 3]
```diff
@@ -0,0 +1,2512 @@
...
+                    <p>This institution, lately established, is directed by the Sisters of St.
+                        Charles Barrameo.</p>
+                    <p>It is situated in a quiet and healthy part of the City near Helouan Railway
+                        Station, and is accessible by the Nasrieh and Old Cairo tram lines.</p>
+                    <p>The course of studies comprises the English, German, and French languages,
+                        all branches of useful knowledge, needlework, music and painting.</p>
+                    <p>Terms very moderate.</p>
+                    <p>For further information apply to SISTER SUPERIOR, ST. CHARLES' SCHOOL, 11,
+                        Sharia-el-Kassed, Cairo.</p>
+                    <p>24815-3-2</p>
```
Comment (whanley) [inline 1904-10-31.xml:1566]: These numbers, which are part of the paper's accounting system, can be tagged `<measure type="indexNo">`

File: 1904-10-31.xml [hunk 4]
```diff
@@ -0,0 +1,2512 @@
...
+                        necessities of the situaution, in view of the very large amount of funds now
+                        placed at its disposal. We could have wished that agriculture ahould have
+                        received a larger share of administrative attention than heretofore, as for
+                        years past we have persistently advocated in these columns that expediency
+                        of giving effect to its claims. Afterall, husbandry in Egypt is an industry
+                        which in magnitude and importance immeasurably transcends all other. The
+                        expenditure, therefore, connected with the establishment of a department
+                        exclusively devoted to the fostering and development of its interests would
+                        be money profitably employed. "Agriculture" in its purely technical sense is
+                        restricted</p>
```
Comment (whanley) [inline 1904-10-31.xml:1645]: You don't need the paragraph or div breaks. you can just say `restricted <cb n="6"/> to the`

File: 1904-10-31.xml [hunk 5]
```diff
@@ -0,0 +1,2512 @@
...
+                    <head>WINDSOR HOTEL</head>
+                    <p>Largest 1st class and most comfortable Hotel in Alexandria. Facing the sea
+                        Central position. Under English managemnt.</p>
+                    <p>81-11-904</p>
+                </div>
+            </div>
+            <pb n="3"/>
+            <div type="page" n="3" facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p3.jpg"> 
+                <cb n="1"/>
+                <div type="section" feature="wire">
```
Comment (whanley) [inline 1904-10-31.xml:1740]: this section is complex to format. Treat cables as individual divs (type="cable"), each with its own <dateline> and <title>. This section is sometimes spread over two pages; when this is the case, you can use the next and prev attributes. Take a peak at a finished file for an example.

File: 1904-10-31.xml [hunk 6]
```diff
@@ -0,0 +1,2512 @@
...
+                <div type="item">
+                    <head>THE KEDIVIAL YACHT</head>
+                    <p>Mahroussa came out of dry-dock yesterday morning, after being inspected by
+                        the engineer, specially sent out from England for the purpose. The latter
+                        has made out his report on the repais needed by the yacht and sent it to the
+                        Ministry of Finance, for the necessary credit to be opened. The mahroussa is
+                        to be sent to England to be overhauled next summer.</p>
+                </div>
+                </div>
+                <div type="item">
```
Comment (whanley) [inline 1904-10-31.xml:1938]: feature="municipality"

File: 1904-10-31.xml [hunk 7]
```diff
@@ -0,0 +1,2512 @@
...
+                <div type="advert" xml:id="deg-ad-ghb01">
+                    <head>GRAND HOTEL BONNARD, ALEXANDRIA.</head>
+                    <p>2s. PER DAY. WINE INCLUDED.</p>
+                    <p><measure type="indexNo">22855-16.7</measure></p>
+                </div>
+                <cb n="3"/>
+                <div type="item">
+                    <head>SERIOUS AFFRAY AT ASSOUAN.</head>
+                    <p>FIGHT AMONG BISHAREEN.</p>
+                    <p>(FROM OUR OWN CORRESPONDENT).</p>
```
Comment (whanley) [inline 1904-10-31.xml:1978]: `<byline>` (which will require you to make the line above a `<head>`)

File: 1904-10-31.xml [hunk 8]
```diff
@@ -0,0 +1,2512 @@
...
+                    <p>Assouan, Saturday.– A very serious fight occurred here a few days ago amongst
```
Comment (whanley) [inline 1904-10-31.xml:1979]: make the first two words `<dateline>`

File: 1904-10-31.xml [hunk 9]
```diff
@@ -0,0 +1,2512 @@
...
+                        such beautiful boxes of chrysanthemums as thise sent by Mr. Rolo, and
+                        amongst his collection was the best chrysanthemum in the show. The feature
+                        of the large groups were the masses of white bloom. All the groups showed
+                        much better taste than in the earlier exhibitions. The flowers were arranged
+                        by masses of colours, as they are in Europe, and there was not that mixture
+                        of colour which was far from artistic in the earlier shows. The Municipality
+                        showed some very fine blooms, but theywere not for competition. The show was
+                        well attended on both days. The following is the prize-list:– </p>
+                    <p>
+                        <list type="ordered">
```
Comment (whanley) [inline 1904-10-31.xml:2153]: Nice use of list! Impressive!

File: 1904-10-31.xml [hunk 10]
```diff
@@ -0,0 +1,2512 @@
...
+                            <item>24 Chrysanthemums (12 varieties at least). Best chrysanthemum in
+                                the show to be selected by the judges from the boxes exhibited.–
+                                Silver Medal, Prince Omar Toussoun; Bronze Medal, M.
+                                Zervudachi.</item>
+                        </list>
+                    </p>
+                </div>
+                <cb n="5"/>
+                <div type="item">
+                    <p>10. 12 Chrysanthemums (mixed).– 1st, M. Jacques Rolo; 2nd, Prince Omar
```
Comment (whanley) [inline 1904-10-31.xml:2174]: except that the list gets interrupted. no need for paragraph and div to break up; just drop the <cb/> where it's needed in the middle of the list.

File: 1904-10-31.xml [hunk 11]
```diff
@@ -0,0 +1,2512 @@
...
+                    <p>Mlle L.J. Perkin, M.A.T.L. Levy, Mme Delta, M.U. Foeliex, M. Ed.N. Goar, M.F.
+                        Austen, Miss M. Neilson, Mlle Mitzi Valeti, Mlle Fournier, M. Selina Debbas,
+                        Mr. John Simpson, M.A. Peristiani, 9 passagers de 3me classe et 27 de 4me
+                        classe.</p>
+                </div>
+            </div>
+            <pb n="4"/>
+            <div type="page" n="4" facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p4.jpg"> 
+                <cb n="1"/>
+                <div type="item">
```
Comment (whanley) [inline 1904-10-31.xml:2367]: feature

**Review by whanley — COMMENTED** [review]
Really great work--well done. I've made a few minor comments, which should be easy to tweak. I'll merge it once you've addressed them, but I'll already call this complete.

**Comment by JuliaCavaliere** [comment]
Deliverable 2 plus fixes to pages 1-3

**Comment by JuliaCavaliere** [comment]
November 2nd issue only has six pages. This is because it only had six pages for whatever reason. I promise I'm not lazy

File: 1904-11-02.xml [hunk 12]
```diff
@@ -0,0 +1,4678 @@
...
+                    <div type="item">
+                        <head>COAL FOR THE FLEET.</head>
+                        <div type="cable">
+                            <dateline>LAS PALMAS, November 1.</dateline>
+                            <p>Four German steamers laden with coal for the Russians left for the
+                                Cameroons, and three British colliers for Reunion Island.
+                                (Reuter.)</p>
+                        </div>
+                    </div>
+                </div>
```
Comment (whanley) [inline 1904-11-02.xml:1991]: remove this `</div>`, so that the following item is included in the telegrams section

File: 1904-11-01.xml [hunk 13]
```diff
@@ -0,0 +1,5261 @@
...
+                        <head>BECK &amp; CO'S PILSENER BEER</head>
+                        <p>BREMEN.</p>
+                    <p>THE WORLD-WIDE KNOWN THE BEST.</p>
+                        <p>Obtainable from every Respectable Firm In Cairo, Alexandria &amp; the Sudan.</p>
+                        <p>Otherwise apply to</p>
+                        <p>V. J. FLEURENT, Cairo</p>
+                        <p>F. MICALLEF, Sole Agent, 11 Bab Midan, Alexandria</p>
+                    </div>
+                <cb n="2"/>
+                <div type="item">
```
Comment (whanley) [inline 1904-11-01.xml:1964]: the following items need to be in the wire section, and there's a bit more tidying to be done.

**Review by whanley — APPROVED** [review]
This is great work. There are a few things to be tidied up, but I'm going to merge it and you can send a new request with the small changes that remain.

**Pull request merged**